examples/evidently_drift_detection/steps/drift_analyzer/drift_analyzer_step.py

```diff
@@ -10,13 +10,15 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
+from evidently.model_profile import Profile  # type: ignore
+
 from zenml.steps import step
 
 
 @step
 def analyze_drift(
-    input: dict,
+    profile: Profile,
 ) -> bool:
     """Analyze the Evidently drift report and return a true/false value
     indicating whether data drift was detected."""
-    return input["data_drift"]["data"]["metrics"]["dataset_drift"]
+    return profile.object()["data_drift"]["data"]["metrics"]["dataset_drift"]
```

Comment:
they also have this `as_dict()` method? not sure if that's the same thing as the `object()` method...

Reply:
`as_dict()` was removed in the new release, probably replaced with `object()` which does the same thing